pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -366,6 +428,53 @@ class CreateVM extends React.Component {
                            onChange={this.onChangedEventChecked.bind(this, 'startVm')} />
                     {_("Immediately Start VM")}
                 </label>
+
+                <hr />
+
+                { this.props.providerName == 'LibvirtDBus' &&
+                this.state.sourceType != EXISTING_DISK_IMAGE_SOURCE &&
```

Comment:
I believe the whole this option group should go next to the storage size. I like how virt-manager has them grouped. I would go with something similar.
![storage](https://user-images.githubusercontent.com/14921356/55787757-42ef3a00-5ab7-11e9-89af-e2b09dfe80ab.png)

Reply:
After discussion with @garrett , he opted to put this option group at the bottom, as it is optional option group. What do you think @garrett ?